target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8065-r7800.dts

```diff
@@ -165,6 +180,13 @@
 					output-high;
 				};
 			};
+
+			usbvcc_pins: usbvcc_pins {
+				pins = "gpio15", "gpio16", "gpio68";
+				function = "gpio";
+				drive-strength = <2>;
+				bias-pull-up;
```

Comment:
This one is 

	.drv = GPIOMUX_DRV_12MA,
	.pull = GPIOMUX_PULL_DOWN,
	.dir = GPIOMUX_OUT_HIGH,